src/pages/commits/commitChart/ChartToggle.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { RadioGroup, Radio } from "@leafygreen-ui/radio-group";
+import { Label } from "@leafygreen-ui/typography";
+import { useLocation, useHistory } from "react-router-dom";
+import { ChartToggleQueryParams, ChartTypes } from "types/commits";
+import { queryString } from "utils";
+
+const { gray } = uiColors;
+const { stringifyQuery } = queryString;
+
+export const ChartToggle: React.FC<{
+  currentChartType: ChartTypes;
+}> = ({ currentChartType }) => {
+  const { pathname } = useLocation();
+  const { replace } = useHistory();
+
+  const onChangeChartType = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ): void => {
+    const nextChartType = event.target.value as ChartTypes;
```

Comment:
I think we can omit `as ChartTypes`. Do you know why we might need it?

Reply:
Just removed it. I'm not sure why exactly. I followed the example of `onChangeLogType()` in `LogTypes.tsx`.